source/persistency/HDF5Writer.cc

```diff
@@ -215,3 +219,15 @@ void HDF5Writer::WriteStep(int evt_number,
 
   istep_++;
 }
+
+void HDF5Writer::WriteChargeDataInfo(int evt_number, unsigned int sensor_id, unsigned int time_bin, unsigned int charge)
```

Comment:
The sensor id in this case would be the wire id?

Reply:
You can regard a wire as a sensor, since it detects something (in this case, charge).